### PR TITLE
Updating getting started doc 

### PR DESCRIPTION
**Pull request by rbitia**
Adding more pre-req instructions for the azure env and cmdline tools needed.

Using make to create clusters rather than the clusterctl cli.

### CODE REVIEW

**Comment by justaugustus** [comment]
/assign
/ok-to-test

@rbitia -- when you have a chance, please apply for [membership to kubernetes-sigs](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E).

File: docs/getting-started.md [hunk 1]
```diff
@@ -27,12 +27,13 @@
...
-- A set of Azure credentials sufficient to bootstrap the cluster (an Azure service principal with Collaborator rights).
+- A [Microsoft Azure account](https://azure.microsoft.com/en-us/)
+- Install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- Install the [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
 - [KIND]
-- [kubectl]
 - [kustomize]
 - make
-- gettext (with `envsubst` in your PATH)
+- gettext (with `envsubst` in your PATH):
```
Comment (CecileRobertMichon) [inline docs/getting-started.md]: why the `:` ?

File: docs/getting-started.md [hunk 2]
```diff
@@ -238,18 +248,9 @@ I0324 23:54:00.260254   27739 kind.go:72] Ran: kind [delete cluster --name=clust
...
-The kubeconfig for the new cluster is created in the directory from where the above `clusterctl create` was run.
-
-Run the following command to point `kubectl` to the kubeconfig of the new cluster:
-```bash
-export KUBECONFIG=$(pwd)/kubeconfig
-```
+## Next steps
 
-Alternatively, move the kubeconfig file to a desired location and set the `KUBECONFIG` environment variable accordingly.
+You've just created an initial management cluster with Kind. Youa are now ready to set up this management cluster and use it to create workload clusters. Follow the documentation in the [Cluster API Book](https://cluster-api.sigs.k8s.io/tasks/installation.html) to learn more.
```
Comment (CecileRobertMichon) [inline docs/getting-started.md]:
```suggestion
You've just created an initial management cluster with Kind. You are now ready to set up this management cluster and use it to create workload clusters. Follow the documentation in the [Cluster API Book](https://cluster-api.sigs.k8s.io/tasks/installation.html) to learn more. 
```

File: docs/getting-started.md [hunk 3]
```diff
@@ -45,36 +46,50 @@
 [brew]: https://brew.sh/
 [go]: https://golang.org/dl/
 [jq]: https://stedolan.github.io/jq/download/
-[kind]: https://sigs.k8s.io/kind
-[kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+[KIND]: https://sigs.k8s.io/kind
```
Comment (CecileRobertMichon) [inline docs/getting-started.md]:
```suggestion
[kind]: https://sigs.k8s.io/kind
```

File: docs/getting-started.md [hunk 4]
```diff
@@ -45,36 +46,50 @@
 [brew]: https://brew.sh/
 [go]: https://golang.org/dl/
 [jq]: https://stedolan.github.io/jq/download/
-[kind]: https://sigs.k8s.io/kind
-[kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
```
Comment (CecileRobertMichon) [inline docs/getting-started.md]: why remove kubectl?

**Comment by CecileRobertMichon** [comment]
@rbitia does this PR fix #338 ?

**Comment by rbitia** [comment]
Yes it should fix #338!

File: docs/getting-started.md (hunk 4 above)
Comment (rbitia) [reply docs/getting-started.md]: I think I added it above as "install the Kubernetes CLI"

**Pull request merged**